pcb/rules/F5_3.py

```diff
@@ -194,14 +196,16 @@ def check(self):
                 self.bad_grid.append(graph)
                 continue
 
-            x1 = mmToMicrons(start['x'])
-            y1 = mmToMicrons(start['y'])
-
-            x2 = mmToMicrons(end['x'])
-            y2 = mmToMicrons(end['y'])
-
+            # make a list of all x and y coordinates of this graphical elements
+            x1 = mmToNanoMeter(start['x'])
+            y1 = mmToNanoMeter(start['y'])
+            x2 = mmToNanoMeter(end['x'])
+            y2 = mmToNanoMeter(end['y'])
             check = [x1, y2, x2, y2]
 
+            # use a modulo division do check if those coordinates are on the grid
+            # if at least one of the coordinates is not on the grid, add this
+            # element to the bad_grid list
             grid_error = False
             for c in check:
                 if not (c % GRID) == 0:
```

Comment:
Sorry if I'm just not aware of some Python implicit boolean conversion quirks or I'm not getting the "Python way" of doing things, but this looks like an overly complicated way of expressing `if c % GRID:` to me.
Same (to a lesser extent) with the explicit comparisons of len() to zero below.

Reply:
True, that seems weird. But that is not something I want to address as part of this PR. 
Those checks are were not introduced by me and I don't really feel responsible for them. At least they are out of scope for this PR.
The main goal here is to fix my fuckup of #334 which is breaking travis-ci in production.

I plan to set up some kind of CI for the check-scripts after we migrated this repo to gitlab. After that I am open to cleanup and refactoring.

Reply:
No problem. I'm just trying to wrap my head around all that Python madness (`not` instead of `!`, `<space>` instead of `<tab>` and whatnot ;) .)